adapters/smartx/smartx_test.go

```diff
@@ -0,0 +1,24 @@
+package smartx
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const testsDir = "smartxtest"
+const testsBidderEndpoint = "http://localhost/prebid_server"
```

Comment:
Requesting to use endpoint mentioned in static/bidder-info/smartx.yaml. This ensures adapter codes works with  actual endpoint